server/repository/src/migrations/v1_06_00/indexes.rs

```diff
@@ -0,0 +1,56 @@
+use crate::{migrations::sql, StorageConnection};
+
+pub(crate) fn migrate(connection: &StorageConnection) -> anyhow::Result<()> {
+    sql!(
+        connection,
+        r#"
+        CREATE INDEX index_temperature_log_datetime ON temperature_log (datetime);        
```

Comment:
Do we create all indexes on every migration?
Seems a bit heavy handed, and difficult to see what's changed...

Reply:
Not your fault of course but thought I'd add a comment :)

Reply:
Sorry now I see Andrei's comment, looks like these are all being added although not related to temperature logs...

Reply:
yeah - I had the minimum required for the cold chain work, which is what the issue was for. Then Andrei suggested catching up on other indexes which have been missed previously, so I renamed the migration to be more general, but have kept the issue the same.

The usual expectation would be to create indexes when tables or relations are created yeah.